Libplanet.Explorer/Store/MySQLRichStore.cs

```diff
@@ -412,5 +704,43 @@ private void Insert(string tableName, IReadOnlyDictionary<string, object> data)
                 }
             }
         }
+
+        private void Update<T>(
+            string tableName,
+            IReadOnlyDictionary<string, object> data,
+            string key,
+            T value)
+        {
+            using QueryFactory db = OpenDB();
+            try
+            {
+                db.Query(tableName).Where(key, value).Update(data);
+            }
+            catch (MySqlException e)
+            {
+                if (e.ErrorCode == MySqlErrorCode.DuplicateKeyEntry)
+                {
+                    Log.Debug($"Ignore DuplicateKeyEntry in {tableName}");
```

Comment:
What case which has duplicated keys?

Reply:
@moreal 
Actually, the [Update](https://github.com/planetarium/libplanet-explorer/pull/165/files#diff-3e6c1427b9b3c3d3db323726233a0ce84120721145b6572b94f15769855989e9R835-R857) and [Insert](https://github.com/planetarium/libplanet-explorer/pull/165/files#diff-3e6c1427b9b3c3d3db323726233a0ce84120721145b6572b94f15769855989e9R754-R781) methods are no longer being used. Is it ok to delete these entirely? 🤔